HPHBhelper/hogwarts/models.py

```diff
@@ -0,0 +1,37 @@
+from django.db import models
+
+# Create your models here.
+class Games(models.Model):
+    shortcode = models.CharField(max_length=4)
+    isActive = models.BooleanField(default=False)
+    host = models.ForeignKey(Players)
+
+    class TurnOrder(models.Model):
```

Comment:
this could potentially go in the model or in the view, depending the logic, but I would not expect a class in a class here, maybe remove it for now